### PR TITLE
Additional test clean-up changes - manager31

### PR DESCRIPTION
**Pull request by 0rnela**

### CODE REVIEW

File: features/step_definitions/command_steps.rb [hunk 1]
```diff
@@ -183,7 +183,7 @@
   raise 'Previous command must fail, but has NOT failed!' if $fail_code.zero?
 end
 
-When(/^"(.*)" exists on the filesystem of "(.*)"$/) do |file, host|
+When(/^File "(.*)" exists on "(.*)"$/) do |file, host|
```
Comment (MalloZup) [inline features/step_definitions/command_steps.rb]: here is typo on your code.

FIle != file

File: features/min_salt_states_catalog.feature [hunk 2]
```diff
@@ -40,7 +40,7 @@ Feature: Salt states catalog
     Then I should see a "1 Changes" text
     And I click on the css "button#save-btn"
     And I click on the css "button#apply-btn"
-    Then "/root/foobar" exists on the filesystem of "sle-minion"
+    Then file "/root/foobar" exists on "sle-minion"
```
Comment (MalloZup) [inline features/min_salt_states_catalog.feature]: typo

**Comment by Bischoff** [comment]
As Ornela is sick, I'm finishing this PR

File: features/min_salt_states_catalog.feature (hunk 2 above)
Comment (Bischoff) [reply features/min_salt_states_catalog.feature]: there was indeed a type ("File" instead of "file"), but not on this line. This line is correct.

**Comment by Bischoff** [comment]
Tests worked (apart from 3 errors outside of Ornela's work). Merging.

**Pull request merged**